### DISCUSSION TITLE
Used emojis in commits

### DISCUSSION THREAD

**Discussion by baltena** [frenck/home-assistant-config#80] [Q&A]
Hey Franck,

Currently I am in the process of adopting your configuration structure in my own home assistant implementation.
I noticed you use specific emojis in your commit messages. As I got my own to-do list and think it is nice to have a quick identifier. So I am very curious how you use them.

So far I have found:
| **Icon** | **Meaning** |
| --- | --- |
| ⬆️  | Update or add-on? |
| 🚑  | Bug |
| 🧹  | Cleanup |
| 🔥 | Drastic cleanup |
| 🗣  | Voice assistant(s) |
| ✨   | Scenes or Add?|
| 🔨  | Scripts? |
| 📚  | Documentation/Readme? |
| 🤖 | Refactor or Entities?|
| 🚜 | (re-)move |
| 👕 | Yaml or Styling?|
| 🎉 | ? Celebrate big updates/accomplishments |
| 🛑 | Disabled something / turned something off |
| ✏️  | ? |
| ⏰ | ? |

Could you elaborate on your use of emojis?

That would be awesome! Thanks in advance!

**Comment by frenck** [comment]
Hehe, I don't have a strict scheme for that. I generally use whatever I feel fitting for that commit.

- 🤖 I often use for automations
- 🔨 Refactoring/changes
- 👕 Code styling
- ✏️ Documentation/code comments/textual change
- ⏰ Did I ever use that? :P haha, could be 🤷 
- 🔥 Removal of a component, integration, functionality
- ⬆️ Upgrade of a dependency, e.g., Home Assistant, external components, or cards.

But yeah, basically, you are on the path. It gives me a visual point of recognition when browsing the commits.

**Reply by niro1987** [reply]
On my own vscode setup I've always been a fan of the [Gitmoji](https://marketplace.visualstudio.com/items?itemName=Vtrois.gitmoji-vscode) addon. Unfortunatly, I've not been able to install it in the vscode addon. Is there a way to add it?
